src/components/table/package.json

```diff
@@ -1,21 +1,19 @@
 {
-  "name": "@ecl/vanilla-component-table",
+  "name": "@ecl/table",
   "author": "European Commission",
   "license": "EUPL-1.2",
   "version": "4.9.0",
   "description": "ECL Table",
-  "style": "table.scss",
-  "sass": "table.scss",
+  "publishConfig": {
+    "access": "public"
+  },
   "main": "table.js",
   "module": "table.js",
+  "style": "table.scss",
   "dependencies": {
-    "@ecl/builder": "4.9.0",
     "@ecl/dom-utils": "4.9.0",
-    "@ecl/resources-icons": "4.9.0",
-    "@ecl/vanilla-layout-grid": "4.9.0"
```

Comment:
it seems we removed this dep, it should still be there but now the package is @ecl/grid